src/.vuepress/config.js

```diff
@@ -40,6 +341,7 @@ let site_config = {
   locales: {
     "/en/": locales.locale_abstract_en,
     "/zh/": locales.locale_abstract_zh,
+    "/ja/": ja,
     "/ko/": locales.locale_abstract_ko,
   },
```

Comment:
Please follow the existing pattern and do not add more than is necessary to this file. These should be loaded by reference, not to local values.